code_schemes/rqa_s08e06.json

```diff
@@ -3,7 +3,70 @@
   "Name": "RQA S08E06",
   "Version": "0.0.0.1",
   "Codes": [
-
+    {
+      "CodeID": "code-368aec3c",
+      "CodeType": "Normal",
+      "DisplayText": "training and seminars",
+      "NumericValue": 1,
+      "StringValue": "training_and_seminars",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-5079d51f",
+      "CodeType": "Normal",
+      "DisplayText": "financial support from government",
+      "NumericValue": 2,
+      "StringValue": "financial_support_from_government",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-f8e3827c",
+      "CodeType": "Normal",
+      "DisplayText": "support from investors/organisations",
+      "NumericValue": 3,
+      "StringValue": "support_from_investors_organisations",
```

Comment:
Can this be changed to: "support_from_investors_or_organisations"

Reply:
Great suggestion, changed